consensus/polybft/fsm_test.go

```diff
@@ -1597,3 +1626,24 @@ func generateValidatorDelta(validatorCount int, allAccounts, previousValidatorSe
 
 	return
 }
+
+func createTestLogForExitEvent(t *testing.T, exitEventID uint64) *types.Log {
+	t.Helper()
+
+	topics := make([]types.Hash, 4)
+	topics[0] = types.Hash(exitEventABI.ID())
+	topics[1] = types.BytesToHash(itob(exitEventID))
+	topics[2] = types.BytesToHash(types.StringToAddress("0x1111").Bytes())
+	topics[3] = types.BytesToHash(types.StringToAddress("0x2222").Bytes())
+	someType := abi.MustNewType("tuple(string firstName, string lastName)")
+	encodedData, err := someType.Encode(map[string]string{"firstName": "John", "lastName": "Doe"})
+	require.NoError(t, err)
+
+	log := &types.Log{
+		Address: contracts.L2StateSenderContract,
+		Topics:  topics,
+		Data:    encodedData,
+	}
+
+	return log
```

Comment:
Just small thing, it would be nicer if this code is just
```
return &types.Log{
		Address: contracts.L2StateSenderContract,
		Topics:  topics,
		Data:    encodedData,
	}
	```

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1064/commits/65f0dd47f4ce915d25db419433c7c0bc7d1d13a9